node.gypi

```diff
@@ -317,7 +317,8 @@
                       '-Wl,--no-whole-archive' ]
     }],
     [ 'OS=="linux" and component != "shared_library"', {
-          'ldflags': [ '-L<(PRODUCT_DIR)/../nw/lib/', '-lnw',
+          'ldflags': [ '-L<(PRODUCT_DIR)/../nw/lib/',
+                      '-lnw',
```

Comment:
Node fails for me at the link stage here:
```shell
/usr/bin/ld: cannot find -lnw: No such file or directory
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```

My guess is using chromium's linker would resolve this? If so please advise how I can do so.